include/data_checks.py

```diff
@@ -0,0 +1,34 @@
+"""
+Column Level Checks
+"""
+COL_CHECKS = [
+    {"time": {
+        "null_check": {"equal_to": 0},
+        "unique_check": {"equal_to": 0}
+    }},
+    {"use_kw": {
+        "null_check": {"equal_to": 0}
+    }},
+    {"gen_kw": {
+        "null_check": {"equal_to": 0}
+    }},
+    {"temperature": {
+        "min": {"geq_to": -20},
+        "max": {"less_than": 99}
+    }},
+    {"humidity": {
+        "min": {"geq_to": 0},
+        "max": {"less_than": 1}
+    }}
+]
+
+"""
+Table Level Checks
+"""
+TABLE_CHECKS = [
+    {"row_count_check": {"check_statement": "COUNT(*) > 100000"}},
+    {"total_usage_check": {"check_statement": "dishwasher + home_office + "
+                           + "fridge + wine_cellar + kitchen + "
+                           + "garage_door + microwave + barn + "
+                           + " well + living_room  <= house_overall"}}
+]
```

Comment:
The operators will be much more efficient if all these checks are just one column and one table operators. I think we should prefer demonstrating good techniques in calling databases over another dynamic mapping example.

Reply:
Thanks for the feedback on this. I moved all column mappings and table checks to the operators and they now execute without dynamic task mapping.